src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/UpdateStaffReferenceProfileTest.java

```diff
@@ -403,6 +404,40 @@ void should_return_update_staff_user_with_status_code_400_invalid_roles() throws
 
     }
 
+    @Test
+    void should_return_reinvite_staff_user_with_status_code_404_profile_doesnot_exist() throws Exception {
+
+        StaffProfileCreationRequest request = caseWorkerReferenceDataClient.createStaffProfileCreationRequest();
+        request.setEmailId("test@test.com");
+        request.setResendInvite(true);
+
+        Map<String, Object> response = caseworkerReferenceDataClient
+                .updateStaffProfile(request,ROLE_STAFF_ADMIN);
+
+        assertThat(response.get("http_status")).isEqualTo("404");
+        String responseBody = (String) response.get("response_body");
+        assertThat(responseBody.contains(INVALID_PROFILE)).isTrue();
+
+    }
+
+    @Test
+    void should_return_reinvite_staff_user_with_status_code_200_profile() throws Exception {
+
+        StaffProfileCreationRequest request = caseWorkerReferenceDataClient.createStaffProfileCreationRequest();
+        userProfilePostUserWireMockForStaffProfile(HttpStatus.CREATED);
+        request.setResendInvite(true);
+
+        caseworkerReferenceDataClient.createStaffProfile(request,ROLE_STAFF_ADMIN);
+        Map<String, Object> response = caseworkerReferenceDataClient.updateStaffProfile(request,ROLE_STAFF_ADMIN);
+
+        assertThat(response).isNotNull();
+        assertThat(response.get("http_status")).isEqualTo("200 OK");
+        Map responseBody = (Map) response.get("body");
+        assertThat(responseBody.get("case_worker_id")).isNotNull();
+
+    }
+
+
```

Comment:
add below tests:
1.  update user details with **is_resend_invite** false and verify user details have been updated
2. update user details with **is_resend_invite** true and verify user details have been updated
3. idam id updated in staff tables when **is_resend_invite** true

Reply:
ok @rajkatla-hmcts 1 and 3 rd condition i will add it now .. but 2 nd condition is not applicable as resend invite true then user details should not be updated confirmed by @kavithagade12 . both resend invite and update should not work together ... Thank you ... fyi @SalomiGeorge

Reply:
Thanks @rajkatla-hmcts added above integration test scenarios ..kindly review ...